### PR TITLE
Feature/create class structure

### PR DESCRIPTION
**Pull request by RobertoLM13**

### CODE REVIEW

File: library-service/src/main/java/com/example/libraryservice/controllers/BookController.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public ResponseEntity<?> createBook(@RequestBody Book book) {
+        return ResponseEntity.ok(bookService.createBook(book));
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<?> removeBook(@PathVariable Long id) {
+        bookService.removeBook(id);
+        return ResponseEntity.ok().build();
+    }
+
```
Comment (joao-cognizant) [inline library-service/src/main/java/com/example/libraryservice/controllers/BookController.java:29]: We could have an endpoint to retrieve the books from a particular author

File: library-service/src/main/java/com/example/libraryservice/controllers/BookController.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    @GetMapping("/{id}")
```
Comment (joao-cognizant) [inline library-service/src/main/java/com/example/libraryservice/controllers/BookController.java:30]: Perhaps also retrieve all the books from a particular publisher

File: library-service/src/main/resources/application.properties [hunk 3]
```diff
@@ -1 +1,19 @@
 
+spring.application.name=library-service
```
Comment (joao-cognizant) [inline library-service/src/main/resources/application.properties:2]: I think it´s more legible if you use the yml format
Add also the server.port propertie

File: library-service/src/main/resources/application.properties [hunk 4]
```diff
@@ -1 +1,19 @@
...
+
+#eureka.client.service-url.defaultZone:http://localhost:8761/eureka
+#eureka.instance.preferIpAddress=true
+
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.show-sql=true
+
+spring.datasource.url=jdbc:h2:mem:test
```
Comment (joao-cognizant) [inline library-service/src/main/resources/application.properties:14]: Change the databasename to library for example

File: library-service/src/main/java/com/example/libraryservice/controllers/BookController.java (hunk 1 above)
Comment (pabloLeonardoLabriola) [reply library-service/src/main/java/com/example/libraryservice/controllers/BookController.java:29]: Great! It'll be backlogged for the next iteration!

File: library-service/src/main/resources/application.properties (hunk 3 above)
Comment (pabloLeonardoLabriola) [reply library-service/src/main/resources/application.properties:2]: Great! It'll be backlogged for the next iteration!

File: library-service/src/main/resources/application.properties (hunk 4 above)
Comment (pabloLeonardoLabriola) [reply library-service/src/main/resources/application.properties:14]: Great! It'll be backlogged for the next iteration!

File: library-service/src/main/java/com/example/libraryservice/controllers/BookController.java (hunk 2 above)
Comment (pabloLeonardoLabriola) [reply library-service/src/main/java/com/example/libraryservice/controllers/BookController.java:30]: Great! It'll be backlogged for the next iteration!

**Pull request merged**